iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -109,12 +109,16 @@ void buildFlowTransformPassPipeline(OpPassManager &passManager) {
   passManager.addNestedPass<FuncOp>(mlir::createCanonicalizerPass());
   passManager.addNestedPass<FuncOp>(createFusionOfTensorOpsPass());
   passManager.addNestedPass<FuncOp>(
-      IREE::Flow::createConvertToFlowTensorOpsPass());
+      IREE::Flow::createConvertToFlowTensorOpsPass(
```

Comment:
Might want to run this after `createResolveShapedTypeResultDimsPass`. That would eliminate some ops that are only used for dims. Some `linalg.fill` might be of this category.